isotovideo

```diff
@@ -104,10 +109,18 @@ sub version () {
     exit 0;
 }
 
-GetOptions(\%options, 'debug|d', 'workdir=s', 'help|h|?', 'version|v') or usage(1);
+GetOptions(\%options, 'debug|d', 'workdir=s', 'color=s', 'help|h|?', 'version|v') or usage(1);
 usage(0) if $options{help};
 version() if $options{version};
 
+my $color = $options{color} // 'auto';
+# Default to no color if there is no terminal
+$color = 'no' if $color eq 'auto' and (not -t STDOUT or not -t STDERR);
```

Comment:
I wonder about this. Do openQA tests still have color recorded in logs? Is that what https://github.com/os-autoinst/openQA/pull/4771 tries to do? Because we should preserve the behaviour.